gcc/testsuite/gcc.target/riscv/rvv/rvv-vget-vset.c

```diff
@@ -0,0 +1,27 @@
+/* { dg-do compile } */
+
+#include <riscv_vector.h>
+
+vint32m1_t vget(vint32m2_t data)
+{
+  return vget_v_i32m2_i32m1(data, 2);
+  /* { dg-error {index 2 is out of range, should be less than 2.} } */
```

Comment:
I got failed for this case, I guess it's because the error line is mis-matched:
```
/scratch1/kitoc/riscv-gnu-workspace/rv64gcv-public/build/build-gcc-newlib-stage2/gcc/include/riscv_vector.h: In function 'vget':^M
/scratch1/kitoc/riscv-gnu-workspace/rv64gcv-public/build/build-gcc-newlib-stage2/gcc/include/riscv_vector.h:264:1: error: index 2 is out of range, should be less than 2.^M
```

And I can pass that after I update the check line to:
```
 /* { dg-error "index 2 is out of range, should be less than 2." "" { target *-*-* } 0 } */
```